examples/flutter_gallery/flutter_gallery.iml

```diff
@@ -3,8 +3,6 @@
   <component name="NewModuleRootManager" inherit-compiler-output="true">
     <exclude-output />
     <content url="file://$MODULE_DIR$">
-      <excludeFolder url="file://$MODULE_DIR$/.pub" />
```

Comment:
I think `.pub` is an implementation detail for both `pub` and `flutter packages` and should be excluded.
@devoncarew might know for sure.